intTests/test_mir_unsafe_assume_spec_statics/test.saw

```diff
@@ -0,0 +1,55 @@
+enable_experimental;
```

Comment:
The second test case, which _does_ involve mutable statics. I kept this separate to avoid having to clutter the pre-/post-conditions of many specifications with extra statements involving the value of `A`.